spec/models/reducer_spec.rb

```diff
@@ -430,4 +430,45 @@ def reduce_into(extracts, reductions=nil)
       expect(result.to_a).to include(ur)
     end
   end
+
+  it 'avoids the tess user skill bug' do
```

Comment:
Can we be slightly more general describing this spec? I don't mind the comments below, but this it-do should be more usefully named for later.